lib/models/store_product_wrapper.dart

```diff
@@ -46,6 +49,10 @@ class StoreProduct with _$StoreProduct {
     @JsonKey(name: 'subscriptionOptions', nullable: true)
         List<SubscriptionOption>? subscriptionOptions,
 
+    // Offering identifier the store product was presented from
```

Comment:
i added a line "Null if not using offerings or if fetched directly from store via getProducts" in native